packages/actions/test/unit/files.test.ts

```diff
@@ -0,0 +1,165 @@
+                )
+            }
+        })
+    })
+    describe("deleteDir", () => {
+        it("should delete a directory", () => {
+            checkAndMakeNewDirectoryIfNonexistent(existingDirName)
+            deleteDir(existingDirName)
+            expect(directoryExists(existingDirName)).to.be.false
+        })
+    })
+    describe("checkAndMakeNewDirectoryIfNonexistent", () => {
+        it("should create a directory", () => {
+            checkAndMakeNewDirectoryIfNonexistent(existingDirName)
+            expect(directoryExists(existingDirName)).to.be.true
+        })
+        it("should fail to create a directory in a non-existent directory", () => {
+            expect(() => checkAndMakeNewDirectoryIfNonexistent(`${nonExistingDirName}/newDir/`)).to.throw(
+                `ENOENT: no such file or directory, mkdir '${nonExistingDirName}/newDir/'`
+            )
+        })
+    })
+    describe("cleanDir", () => {
+        it("should clean a directory", () => {
+            checkAndMakeNewDirectoryIfNonexistent(existingDirName)
+            cleanDir(existingDirName)
+            expect(directoryExists(existingDirName)).to.be.true
+        })
+        it("should create a new directory if trying to clean a non-existent directory", () => {
+            expect(cleanDir(nonExistingDirName)).to.be.undefined
+            expect(directoryExists(nonExistingDirName)).to.be.true
+        })
+        // clean up after test
+        afterAll(() => {
+            deleteDir(nonExistingDirName)
+        })
+    })
+    describe("readJSONFile", () => {
+        const filePathForReadingJSON = `${existingDirName}/test.json`
+        it("should read a JSON file", () => {
+            writeLocalJsonFile(filePathForReadingJSON, fileContentJSON)
+            expect(readJSONFile(filePathForReadingJSON)).to.deep.equal(fileContentJSON)
+        })
+        it("should fail to read a JSON file from a non-existent directory", () => {
+            expect(() => readJSONFile(`${nonExistingDirName}/test.json`)).to.throw("File not found")
+        })
+        afterAll(() => {
+            deleteDir(filePathForReadingJSON)
+        })
+    })
+    describe("writeLocalJsonFile", () => {
+        it("should write a JSON file", () => {
+            const filePathForWritingJSON = `${existingDirName}/test.json`
+            writeLocalJsonFile(filePathForWritingJSON, fileContentJSON)
+            expect(readJSONFile(filePathForWritingJSON)).to.deep.equal(fileContentJSON)
+        })
+        it("should fail to write a JSON file to a non-existent directory", () => {
+            const filePathForWritingJSON = `${nonExistingDirName}/test.json`
+            expect(() => writeLocalJsonFile(filePathForWritingJSON, fileContentJSON)).to.throw(
+                `ENOENT: no such file or directory, open '${filePathForWritingJSON}'`
+            )
+        })
+    })
+    describe("downloadFileFromUrl", () => {
+        const url =
```

Comment:
This file is about 70MB. Could we shot for a smaller one? (around 1MB) to avoid bandwidth usage when testing.

I currently have a very slow connection and test didn't pass in 60 seconds.